dependencies/Gemfile.lock

```diff
@@ -53,11 +91,12 @@ PLATFORMS
   ruby
 
 DEPENDENCIES
+  cfn-nag (~> 0.7.12)
   rubocop (~> 1.6.1)
   rubocop-github (~> 0.16.1)
   rubocop-performance (~> 1.7.1)
   rubocop-rails (~> 2.7)
   rubocop-rspec (~> 2.2.0)
 
 BUNDLED WITH
-  2.2.11
+   2.1.4
```

Comment:
No idea how ruby/gemfiles work, but wonder why it looks like the version got downgraded here.

Reply:
Yes I just did normal bundle command for creating lock file.